Cargo.lock

```diff
@@ -1970,7 +1987,7 @@ dependencies = [
 [[package]]
 name = "libusb-sys"
 version = "0.2.4"
-source = "git+https://github.com/paritytech/libusb-sys#1d33d9840a82adaf4d6a1a0f5141f022e5676802"
+source = "git+https://github.com/paritytech/libusb-sys#4418e92c82509d1f049f7942dffe62a0bf5dd871"
 dependencies = [
```

Comment:
hrm, is this change intentional?

Reply:
Nope, let me restore `Cargo.lock` from master.